### ISSUE TITLE
Error trying to get user's videos

### ISSUE THREAD

**Issue by nadavkav** [justinhunt/moodle-tinymce_youtube#1]
Hi Justin,
I am getting the following error on a Moodle 2.8 version, I have, for the first time, installed youtube anywhere onto.

Running quick xdebug, I can see the initialization of the $yt is OK.
And the following:
      $vidfeed= "https://gdata.youtube.com/feeds/api/users/default/uploads";
      $videoFeed = $yt->getVideoFeed($vidfeed);
Does not work.
Did they change API?
Should I get an updated Zend_GData lib?

---

line 700 of /lib/zend/Zend/Gdata/App.php: Zend_Gdata_App_HttpException thrown
line 219 of /lib/zend/Zend/Gdata.php: call to Zend_Gdata_App->performHttpRequest()
line 861 of /lib/zend/Zend/Gdata/App.php: call to Zend_Gdata->performHttpRequest()
line 754 of /lib/zend/Zend/Gdata/App.php: call to Zend_Gdata_App->get()
line 205 of /lib/zend/Zend/Gdata/App.php: call to Zend_Gdata_App->importUrl()
line 162 of /lib/zend/Zend/Gdata.php: call to Zend_Gdata_App->getFeed()
line 263 of /lib/zend/Zend/Gdata/YouTube.php: call to Zend_Gdata->getFeed()
line 427 of /lib/editor/tinymce/plugins/youtube/youtubelib.php: call to Zend_Gdata_YouTube->getVideoFeed()
line 54 of /lib/editor/tinymce/plugins/youtube/browser.php: call to youtubeapi->fetch_youtube_browselist()

**Comment by nadavkav** [comment]
Ping
Any updates on this?

**Comment by justinhunt** [comment]
No updates sorry. Nadav I have not been able to look at this plugin for about 2 years. To be honest I can't see the time to do this suddenly arriving. Apologies.

**Comment by nadavkav** [comment]
Thank you very much Justin for your quick reply.
I guess I will have to dig into this myself :-)
Seems like Google changed their YouTube API, again.
I will start with updating the library code and see where it goes from there...
Thanks!

**Comment by justinhunt** [comment]
My gut feeling is that this is a bit of a rabbit hole, hopefully its not though. And I will help if I can.

One of the reasons this is unlikely to get much dev. time is that I believe it would be best to rethink things and work with Moodle's Atto editor.

**Comment by nadavkav** [comment]
Indeed, going forward with ATTO is the best. but I still have this one old Moodle system that is using TinyMCE (as ATTO have some RTL issues and they can not switch easily)

btw, off topic, I just used your amazing generico filter with mermaid. super cool and easy to integrate. what a time saver. bless you!

**Comment by justinhunt** [comment]
Glad you like Generico/Mermaid. Mermaid is really nifty. Its hard to keep html rubbish from the editor getting in and screwing up the mermaid code, so it needs its own atto plugin I reckon. A good project for someone I guess.
